src/executor/plan_executor.cpp

```diff
@@ -376,6 +376,11 @@ executor::AbstractExecutor *BuildExecutorTree(
       child_executor = new executor::DropExecutor(plan, executor_context);
       break;
 
+    case PlanNodeType::ANALYZE:
+      LOG_TRACE("Adding Analyze Executer");
```

Comment:
Just find the typo was copied from other log traces in the same switch block, haha. We will fix others too.
See:
https://github.com/cmu-db/peloton/blob/master/src/executor/plan_executor.cpp#L303
https://github.com/cmu-db/peloton/blob/master/src/executor/plan_executor.cpp#L308